src/components/EditContentModal/EditContentModal.tsx

```diff
@@ -234,15 +234,11 @@ const EditContentModal = ({ values, open, setClosed }: EditContentProps) => {
     <Modal
       variant={ModalVariant.medium}
       title='Edit custom repository'
+      ouiaId='edit_custom_repository'
       help={
         <Popover
           headerContent={<div>Help Popover</div>}
-          bodyContent={
-            <div>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam id feugiat augue, nec
-              fringilla turpis.
-            </div>
-          }
+          bodyContent={<div>Use this form to edit the values of an existing repository.</div>}
           footerContent='Popover Footer'
```

Comment:
```suggestion
          footerContent='* mandatory fields'
```

Reply:
Capitalize first letter

Reply:
@avisiedo agreed we can just remove this